utility/src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.util.utility
```

Comment:
This seems like something that should stay under util/networking.

We generally want to avoid generic buckets like utility, and it's a bit extra confusing to have utility/utility. We try to keep utility subpackages very specific for this reason.